demobot.py

```diff
@@ -44,15 +44,15 @@ async def on_ready():
 @bot.event
 async def on_command_error(ctx, error):
     if isinstance(error, commands.errors.CheckFailure):
-        await ctx.send('You do not have the correct role for this command.')
+        await ctx.send('You do not have the correct role for this command')
     elif isinstance(error, commands.errors.MissingRequiredArgument):
-        await ctx.send('You are missing a required argument.')
+        await ctx.send('You are missing a required argument')
```

Comment:
you are removing a full stop here?

Reply:
don't remember changing this at all... weird